src/System.Text.Json/tests/Utf8JsonWriterTests.cs

```diff
@@ -3097,6 +3148,17 @@ public void EscapeSurrogatePairs(bool formatted, bool skipValidation)
             }
         }
 
+        [Fact]
+        public void EscapeSurrogatePairsThrowsHighSurrogateMissing()
+        {
+            string propertyName = "a \uD800 \uDC00\uDE6D a";
```

Comment:
Why are there space character in here? The test name says "High Surrogate Missing", but it looks like the writer will fail with low surrogate missing because it will try to write these two characters: \uD8000, SPACE, and the surrogate pair ends up missing the low surrogate following it (\uDC00).